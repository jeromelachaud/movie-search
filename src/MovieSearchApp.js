import React, { Component } from 'react';
import SearchForm from './component/SearchForm';
import './MovieSearchApp.css';
class MovieSearchApp extends Component {
  render() {
    return (
      <div className="MovieSearchApp">
        <SearchForm />
      </div>
    );
  }
}

export default MovieSearchApp;
