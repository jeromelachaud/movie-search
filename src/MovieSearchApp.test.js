import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MovieSearchApp from './MovieSearchApp';

describe('MovieSearchApp', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MovieSearchApp />, div);
  });

  it('should render as expected', () => {
    const movieSearchAppComponent = shallow(<MovieSearchApp />);
    expect(toJson(movieSearchAppComponent))
      .toMatchSnapshot();
  });
});
