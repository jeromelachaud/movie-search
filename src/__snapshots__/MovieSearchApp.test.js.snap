// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieSearchApp should render as expected 1`] = `
<div
  className="MovieSearchApp"
>
  <SearchForm />
</div>
`;
